rest_registration/signals.py

```diff
@@ -9,3 +9,8 @@
 
 # A user has activated his or her account.
 user_activated = Signal(providing_args=["user", "request"])
+
+# A user has verified his or her new email.
+user_changed_email = Signal(
+    providing_args=["user", "new_email", "old_email", "request"]
+)
```

Comment:
Same here, linting issue. It was a single char too long as a single line.

Reply:
Err, just realized I forgot to implement the `new_email`, `old_email`...  will do.

Reply:
Hi, thanks for the changes anyway. I will do one cosmetic change to generalize the code